### PR TITLE
Py

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: py/bitbox02/communication/usart/usart.py [hunk 1]
```diff
@@ -29,8 +29,8 @@ class SerialPort(PhysicalLayer):
     (i.e. to make it implement communication.PhysicalLayer).
     """
 
-    def __init__(self, *args: Any, **kwargs: Any):
-        self.port = serial.Serial(*args, **kwargs)
+    def __init__(self, port: str, timeout: Optional[int] = None):
```
Comment (conte91) [inline py/bitbox02/communication/usart/usart.py]: The timeout is set for each `read` operation explicitely (see the `read` method). If we want to change it to be in the constructor (or remove it completely if you ask me...) we should change the definition of `PhysicalLayer` as well (note that this has been defined to be compatible with the `hid` class, so there's no reason not to adapt both to our needs).

File: py/load_firmware.py [hunk 2]
```diff
@@ -203,7 +203,7 @@ def main() -> int:
         return 1
 
     if args.usart is not None:
-        serial_port = usart.SerialPort(args.usart, 115200, timeout=3)
+        serial_port = usart.SerialPort(args.usart, timeout=3)
```
Comment (conte91) [inline py/load_firmware.py]: For the same reason as above, this stray `timeout` argument should be removed (as it's going to be ignored)

File: py/bitbox02/communication/usart/usart.py (hunk 1 above)
Comment (benma) [reply py/bitbox02/communication/usart/usart.py]: fixed

File: py/load_firmware.py (hunk 2 above)
Comment (benma) [reply py/load_firmware.py]: fixed

**Pull request merged**